compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/SymbolFinder.java

```diff
@@ -283,6 +285,10 @@ public void visit(BLangFunction funcNode) {
             return;
         }
 
+        if (funcNode.getMarkdownDocumentationAttachment() != null) {
+            lookupNodes(funcNode.markdownDocumentationAttachment.parameters);
+        }
```

Comment:
Better if these are also handled in the normal flow of the visitor, instead of adding null checks everywhere and checking.

Reply:
Replaced with `lookupNode()` for `markdownDocumentationAttachment` property.